### PR TITLE
Added a "Supported Versions" table to the downloads page.

### PR DESCRIPTION
**Pull request by timgraham**
policy

### CODE REVIEW

File: djangoproject/templates/releases/download.html [hunk 1]
```diff
@@ -48,9 +48,77 @@
...
+      data loss fixes applied for a guaranteed period of time, typically 3+ years,
+      regardless of the pace of releases afterwards.</p>
+
+    <p>See the <a href="https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions">
+  supported versions policy</a> for detailed guidelines about what fixes will be backported.</p>
+  <table class='django-supported-versions'>
+      <tr>
+        <th>Release Series</th>
+        <th>Latest Release</th>
+        <th>End of mainstream support<sup>1</sup></th>
```
Comment (bmispelon) [inline djangoproject/templates/releases/download.html]: Would there be a way to have `<sup>1</sup>` link to the corresponding note?

Comment (timgraham) [inline djangoproject/templates/releases/download.html]: Yes, but unless you have a very small screen the footnotes and table should fit on the screen so I didn't think it was really useful.

Comment (bmispelon) [inline djangoproject/templates/releases/download.html]: Good point about the size of the table but I wonder if has accessibility implications (I'm not the most knowledgeable in that regard sadly).

Comment (timgraham) [inline djangoproject/templates/releases/download.html]: Well, it's trivial done, so added.

File: djangoproject/templates/releases/download.html [hunk 2]
```diff
@@ -48,9 +48,77 @@
...
+  <h2>Supported Versions</h2>
+  <p>Major releases (A.B, A.B+1, etc.) will happen roughly every six to nine months.
+    These releases will contain new features, improvements to existing features, and such.</p>
+
+  <p>Minor releases (A.B.C, etc.) will be issued as needed, often to fix security issues.
+    These releases will be 100% compatible with the associated major release, unless
+    this is impossible for security reasons or to prevent data loss. So the answer to
+    “should I upgrade to the latest minor release?” will always be “yes.”</p>
+
+    <p>Additionally, the Django team will occasionally designate certain releases
```
Comment (MarkusH) [inline djangoproject/templates/releases/download.html]: Unindent to match previous paragraph.

File: djangoproject/templates/releases/download.html [hunk 3]
```diff
@@ -48,9 +48,77 @@
...
+      to be “Long-term support” (LTS) releases. LTS releases will get security and
+      data loss fixes applied for a guaranteed period of time, typically 3+ years,
+      regardless of the pace of releases afterwards.</p>
+
+    <p>See the <a href="https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions">
```
Comment (MarkusH) [inline djangoproject/templates/releases/download.html]: This as well.

File: djangoproject/templates/releases/download.html [hunk 4]
```diff
@@ -48,9 +48,77 @@
...
+  supported versions policy</a> for detailed guidelines about what fixes will be backported.</p>
+  <table class='django-supported-versions'>
+      <tr>
+        <th>Release Series</th>
+        <th>Latest Release</th>
+        <th>End of mainstream support<sup><sup><a href="#ft1">1</sup></sup></th>
```
Comment (MarkusH) [inline djangoproject/templates/releases/download.html]: Duplicate `<sup></sup>`

**Comment by MarkusH** [comment]
I personally find the red a bit too red-ish and give it a more pastel look, but well, I'm not a designer ;)

**Comment by timgraham** [comment]
Can't say I disagree.

img

**Comment by MarkusH** [comment]
Much better :+1: Now I'm worrying about people with dyschromatopsia.

**Pull request merged**